src/HealthCheckServiceProvider.php

```diff
@@ -4,24 +4,30 @@
 
 use Illuminate\Support\ServiceProvider;
 use UKFast\HealthCheck\Commands\CacheSchedulerRunning;
-use UKFast\HealthCheck\Controllers\PingController;
 use UKFast\HealthCheck\Controllers\HealthCheckController;
+use UKFast\HealthCheck\Controllers\PingController;
 
 class HealthCheckServiceProvider extends ServiceProvider
 {
     public function boot()
     {
         $this->configure();
+
+        $v = $this->app->make('router');
+
+        //dd(get_class($v));
+
         $this->app->make('router')->get($this->withBasePath('/health'), [
             'middleware' => config('healthcheck.middleware'),
-            'uses' => HealthCheckController::class
-        ]);
+            'uses' => HealthCheckController::class,
+        ])->name(config('healthcheck.routename'));
```

Comment:
Following on from my earlier suggestion.

```suggestion
        ])->name(config('healthcheck.route-name'));
```